### PR TITLE
feat: set run title and description with enterpriseStart, closes HYB-593

### PR DESCRIPTION
**Pull request by Obero**
Motivation:
- Users want to identify their simulation runs conditions
- Users should be able to set run title and description to add metadata

Modifications:
- Pass run title and description as start operation parameters

Result:
- Run title and description can be set by users at simulation start

### CODE REVIEW

**Comment by Obero** [comment]
In draft until this PR is released: https://github.com/gatling/gatling-enterprise-plugin-commons/pull/131

Expected commons lib version: 1.9.0

File: src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy [hunk 1]
```diff
@@ -124,6 +144,28 @@ class GatlingPluginExtension {
...
+
+        @Input
+        @Optional
+        String getRunDescription() {
+            if (runDescription == null) {
+                def systemRunDescription = System.getProperty(RUN_DESCRIPTION_PROPERTY)
+                return systemRunDescription ? systemRunDescription : null
+            } else {
+                return runDescription
+            }
```
Comment (Isammoc) [inline src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy:166]:
```suggestion
            return runDescription ?: System.getProperty(RUN_DESCRIPTION_PROPERTY) ?: null
```

**Review by bastien-gatling — APPROVED** [review]

File: src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package io.gatling.gradle
 
+
```
Comment (tpetillot) [inline src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy:3]:
```suggestion
```

**Review by tpetillot — APPROVED** [review]

File: src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy [hunk 3]
```diff
@@ -102,43 +124,58 @@ class GatlingPluginExtension {
...
+        @Input
+        @Optional
+        String getRunTitle() {
+            if (runTitle == null) {
+                def systemRunTitle = System.getProperty(RUN_TITLE_PROPERTY)
+                return systemRunTitle ? systemRunTitle : null
             } else {
-                return simulationId
+                return runTitle
             }
```
Comment (Isammoc) [inline src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy:155]:
```suggestion
        String getRunTitle() {
            return runTitle ?: System.getProperty(RUN_TITLE_PROPERTY) ?: null
```

File: src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy [hunk 4]
```diff
@@ -102,43 +124,58 @@ class GatlingPluginExtension {
...
-                return systemSimulationName ? systemSimulationName : null
+        String getRunDescription() {
+            if (runDescription == null) {
+                def systemRunDescription = System.getProperty(RUN_DESCRIPTION_PROPERTY)
+                return systemRunDescription ? systemRunDescription : null
             } else {
-                return simulationName
+                return runDescription
             }
         }
```
Comment (Isammoc) [inline src/main/groovy/io/gatling/gradle/GatlingPluginExtension.groovy:167]:
```suggestion
        String getRunDescription() {
            return runDescription ?: System.getProperty(RUN_DESCRIPTION_PROPERTY) ?: null
        }
```

**Review by Isammoc — CHANGES_REQUESTED** [review]

**Review by Isammoc — APPROVED** [review]

**Pull request merged**